server/gitUtility/initialSeed.js

```diff
@@ -0,0 +1,54 @@
+require('./../../config/config');
+const { gitService } = require('./gitService');
+const { Stargazers } = require('./../db/stargazers');
+const { Repository } = require('./../db/repositories');
+const { stargazerList } = require('./stargazerList');
+const { logger } = require('./../../config/winston');
+
+// stargazerList = ['Kureev', 'yyx990803'];
+
+function seedStargazerRecords(token) {
+    // Get List from stargazerList.js and update records for stargazer.
+    for (const stargazer of stargazerList) {
+        logger.info(`*****Getting Record for stargazer:${stargazer}*****`);
+        gitService.getUserDetails(stargazer, token).then((user) => {
+            Stargazers.findOneAndUpdate({ login: user.login }, user,
+                                        { new: true, upsert: true },
+                                        (error, doc) => {
+                                            if (error) throw error;
+                                            //logger.debug(`Updated: ${stargazer}`);
```

Comment:
Can you remove this commented code please? Unless it's really necessary for the future.